modules/contentbox/modules/contentbox-admin/layouts/inc/HTMLHead.cfm

```diff
@@ -62,12 +63,11 @@
     <!--- ********************************************************************* --->
     <!---                            JS LIBRARIES                				--->
     <!--- ********************************************************************* --->
-    <cfif getSetting( "environment" ) eq "development">
-        <script defer src="#prc.cbroot#/includes/js/contentbox-pre.js"></script>
-        <script defer src="#prc.cbroot#/includes/js/contentbox-app.js"></script>
-    <cfelse>
-        <script defer src="#prc.cbroot#/includes/js/contentbox-pre.min.js"></script>
-        <script defer src="#prc.cbroot#/includes/js/contentbox-app.min.js"></script>
-    </cfif>
+	<script defer src="#prc.cbroot#/includes/js/runtime.js"></script>
+	<script defer src="#prc.cbroot#/includes/js/vendor.js"></script>
+	<script defer src="#cbadminElixirPath( fileName='js/bootstrap.js' )#"></script>
```

Comment:
Any reason why 3 js files instead of just one?

Reply:
This is Webpack's convention when you compile with elixir.   `runtime.js` always has to come first, then the vendor pack ( which are the extracted module singletons from the `import ..` and `require` statements found in `app.js`, and then the explicitly compiled files.

Reply:
Of note:  I left the `bootstrap` pack alone, mostly, for now.  There's no reason, really, that those can be added as `require` statements in `app.js` though - in which case they would go to `vendor.js`.  I'd like to do that in a separate iteration, though, since I've been able to QA the admin, in full, with this variation.